src/hooks/useGovernanceProfile.ts

```diff
@@ -8,6 +9,10 @@ import { ErrorCategory } from '../utils/errorCategories'
 
 import { DEFAULT_QUERY_STALE_TIME } from './constants'
 
+function isNotFoundOrBadRequest(error: RequestError) {
+  return error.statusCode === 404 || error.statusCode === 400
```

Comment:
Why would this be a bad request?

Reply:
`validateAddress` throws BadRequest errors for invalid addresses (we don't need to report those)